### DISCUSSION TITLE
Search options

### DISCUSSION THREAD

**Discussion by JeremyJaydan** [dailydotdev/daily#294] [Feature Requests]
Some search options similar to Google search could be neat:

- source: \<source url\>
- before: \<date\>
- after: \<date\>
- #\<tag\>
- "\<exact phrase\>"
- -excludeword

**Comment by idoshamun** [comment]
Hey @JeremyJaydan,

Thanks for the suggestion :)
This is a great idea and we'll look into when it's the right time to work on it.
We're now focused on improving content quality and personalization.

**Comment by andypiper** [comment]
Any progress on this?

What's the underlying engine for search - does it offer any syntax that would be helpful for us to optimize queries ourselves?

**Reply by idoshamun** [reply]
Not at the moment, unfortunately, improving the search engine is not yet prioritized. We're more into collaboration now so stay tuned for awesome new features. I hope we may consider it again in a few months

**Comment by faddah** [comment]
I also would like to be able to SORT serach results of articles once they are retrieved (searching for subjects like "Python Big Data," "Machine Learning," "JavaScript Promises," or "AWS Certification") by DATE, ascending (oldest first) or descending (newest first). This should not be that hard a feature to add, as it does not involve changes to your search engine itself, just filtering/sorting through the already delivered results. I would like to see my article results for the most current, latest articles on a tech subject FIRST, not something from 3 - 5 years ago, which could be way out of date, information-wise.

**Comment by stephdotnet** [comment]
I totally agree with the sorting feature. I use the search engine to find posts about a topic, and sometimes i feel the need to sort them by date to be able to retrieve the newest (and most up to date with documentation/practices). 

I see that this feature is "accepted", but i just wanted to add my support to this one :)

Thanks for your work👏👍

**Comment by randy** [comment]
I was about to post a request similar to this, so I'll just add my details here:

> I'd really love a way to search by "author tag." Meaning, if I type `@daily_updates`, I'd like to see all posts from `@daily_updates`. 

P.S. @idoshamun, it's been over a year since your comment. Can we get an update on where this might fall? 😉

**Reply by idoshamun** [reply]
Hi @randy,

Great seeing you here as well. We're about to deprecate our existing search and introduce something else entirely.
Sorry for not communicating it better. We're deep into delivering new things.

**Comment by vastnight** [comment]
if the content is great but we can't access it due to a broken search bar, it should be looked into; but then again alot of modern websites seem to miss this feature for whatever reason...

anyways using google search operators + time filters seems to somewhat work.

**Comment by carliblaz** [comment]
I don't know if I'm the only one, but I really miss search in squads, so if there is a feature request for search, please think about adding search in squads! :)

**Comment by sabdailydev** [comment]
Hey all, we added a few filters to Search results (based on time as well as content types :-) ) ! Sample screenshots are attached. Let us know what you think. 
<img width="572" height="381" alt="Screenshot 2025-07-14 at 10 09 55" src="https://github.com/user-attachments/assets/af453d4c-d711-4a1f-838f-f783bb2b1575" />
<img width="571" height="351" alt="Screenshot 2025-07-14 at 10 10 03" src="https://github.com/user-attachments/assets/71c54b5e-dc57-447e-8af3-689f07fbdc13" />
